src/main.cpp

```diff
@@ -35,8 +36,22 @@ void publishMessage() {
     if(DEBUG_MEM){
         DEBUG_SERIAL_LN("\nFREE RAM: " + String(System.freeMemory()) + "B / 128000B");
     }
+}
 
+// allows us to define behavior depending on publish status
+void publishCallback(String payload, DataQueue::PublishStatus status) {
+    switch (status) {
+        case DataQueue::PublishingAtMaxFrequency:
+            DEBUG_SERIAL_LN("CURRENTLY PUBLISHING AT MAX FREQUENCY");
+            return;
```

Comment:
Isn't the standard way to exit a switch statement to use _break;_? It shouldn't make a difference functionally here, but I think it would be good to keep it consistent.

Reply:
yep we can break out of it